pmd-java/src/main/resources/category/java/multithreading.xml

```diff
@@ -30,7 +30,7 @@ gets it.
 <![CDATA[
 public class Foo {
     // Try to avoid this:
-    synchronized void foo() {
+    synchronized void bar() {
```

Comment:
```suggestion
    synchronized void foo() {
```

foo is the method, that is synchronized. bar is the same method, but with block level synchronization.
Same pattern as below with fooStatic() and barStatic()